tests/admin_scripts/tests.py

```diff
@@ -980,24 +980,43 @@ def test_custom_command_with_environment(self):
         out, err = self.run_manage(args, 'alternate_settings')
         self.assertOutput(
             out,
-            "EXECUTE: noargs_command options=[('no_color', False), "
+            "EXECUTE: noargs_command options=[('force_color', False), ('no_color', False), "
             "('pythonpath', None), ('settings', None), ('traceback', False), "
             "('verbosity', 1)]"
         )
         self.assertNoOutput(err)
 
     def test_custom_command_output_color(self):
-        "alternate: manage.py output syntax color can be deactivated with the `--no-color` option"
+        """alternate: manage.py output syntax color can be deactivated with the
+        `--no-color` and `--force-color` option
+        """
         args = ['noargs_command', '--no-color', '--settings=alternate_settings']
         out, err = self.run_manage(args)
         self.assertOutput(
             out,
-            "EXECUTE: noargs_command options=[('no_color', True), "
+            "EXECUTE: noargs_command options=[('force_color', False), ('no_color', True), "
             "('pythonpath', None), ('settings', 'alternate_settings'), "
             "('traceback', False), ('verbosity', 1)]"
         )
         self.assertNoOutput(err)
 
+        args = ['noargs_command', '--force-color', '--settings=alternate_settings']
+        out, err = self.run_manage(args)
+        self.assertOutput(
+            out,
+            "EXECUTE: noargs_command options=[('force_color', True), ('no_color', False), "
+            "('pythonpath', None), ('settings', 'alternate_settings'), "
+            "('traceback', False), ('verbosity', 1)]"
+        )
+        self.assertNoOutput(err)
+
+        args = ['noargs_command', '--no-color', '--force-color',
+                '--traceback', '--settings=alternate_settings']
+        out, err = self.run_manage(args)
+        self.assertNoOutput(out)
+        msg = 'CommandError: The --no-color and --force-color options cannot be used together'
```

Comment:
I think we can move `msg` to the assertion, I don't see much value in creating a temporary variable here.